contents/docs/cdp/batch-exports/snowflake.md

```diff
@@ -51,3 +51,11 @@ CREATE TABLE IF NOT EXISTS "{database}"."{schema}"."{table_name}" (
 )
 COMMENT = 'PostHog events table'
 ```
+
+> **Note:** This is the same query used by PostHog Batch Exports if the table is not present in your database. Notice that the database, schema, and table_name parameters are surrounded by double quotes to respect the casing provided during configuration.
+
+### File staging
+
+PostHog batch exports for Snowflake use Snowflake's [internal table stages](https://docs.snowflake.com/en/user-guide/data-load-local-file-system-create-stage#table-stages) to stage the files that are copied into your Snowflake tables. We recommend using separate tables to export data from PostHog to avoid conflicting with other workflows that use internal table stages.
```

Comment:
```suggestion
Batch exports use Snowflake's [internal table stages](https://docs.snowflake.com/en/user-guide/data-load-local-file-system-create-stage#table-stages) to stage the files copied into your Snowflake tables. We recommend using separate tables to export data from PostHog to avoid conflicting with other workflows that use internal table stages.
```